recipes/approxposterior/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "approxposterior" %}
+{% set version = "0.1" %}
+{% set sha256 = "83d9559b325ec45765e00b57992c9f6b8378387b45b9455ea22429b6bbce2d1c" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pytest
+    - python
+    - pip
+    - numpy
+    - matplotlib
+    - scipy
+    - george
+    - emcee
+    - corner
+    - scikit-learn
+  run:
+    - python
+    - numpy
+    - matplotlib
+    - scipy
+    - george
+    - emcee
+    - corner
+    - scikit-learn
+
+test:
+  imports:
+    - approxposterior
+    - approxposterior.bp
+
+about:
+  home: http://github.com/dflemin3/approxposterior
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Python implementation of Bayesian Approximate Posterior Estimation algorithm'
+
+  description: |
+    This package is a Python implementation of Bayesian Active Learning for Posterior Estimation by Kandasamy et al. (2015) and Adaptive Gaussian process approximation for Bayesian inference with expensive likelihood functions by Wang & Li (2017). These algorithms allows the user to compute approximate posterior probability distributions using computationally expensive forward models by training a Gaussian Process (GP) surrogate for the likelihood evaluation. The algorithms leverage the inherent uncertainty in the GP's predictions to identify high-likelihood regions in parameter space where the GP is uncertain. The algorithms then run the forward model at these points to compute their likelihood and re-trains the GP to maximize the GP's predictive ability while minimizing the number of forward model evaluations. Check out Bayesian Active Learning for Posterior Estimation by Kandasamy et al. (2015) and Adaptive Gaussian process approximation for Bayesian inference with expensive likelihood functions by Wang & Li (2017) for in-depth descriptions of the respective algorithms.
+  doc_url: https://dflemin3.github.io/approxposterior/
+  dev_url: https://github.com/dflemin3/approxposterior
+
+extra:
+  recipe-maintainers:
+    - dflemin3
+    - jakevdp
```

Comment:
@jakevdp are you ok with being maintainer?

Reply:
Yep!